cmd/controller/sparsefilegenerator.go

```diff
@@ -45,6 +45,10 @@ On Shutdown, the status of the sparse file Disk CR will be marked as Unknown.
 */
 
 const (
+	/*ENV_SPARSE_FILE_DIR - if it specified as a environment variable,
+	a Sparse file with specified size (SPARSE_FILE_SIZE) will
+	be created inside specified directory (ENV_SPARSE_FILE_DIR)
+	and an associated Disk CR will be added to Kubernetes.*/
```

Comment:
@bege13mot  Providing comment for only one const will not fix the lint error. Please provide comments for all, otherwise  it will still give the error for next const i.e. ENV_SPARSE_FILE_SIZE here.

Reply:
Hi @sagarkrsd ,
Thank you for the review. It is fixed.